community/en/tiny_imagenet_custom_resnet.ipynb

```diff
@@ -0,0 +1,552 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
If you want to make use of the added utility funcitons instead of repeating them inline try this:

```
!pip install -q git+https://github.com/tensorflow/examples.git
from tensorflow_examples.models.custom_resnet import custom_resnet
```

ref:

https://www.tensorflow.org/beta/tutorials/generative/cyclegan
https://www.tensorflow.org/beta/tutorials/images/segmentation

Reply:
I intentionally included all the code in the notebook since it used tf.keras all through. I have cleaned it up and will update it in the next commit.